slim/nets/resnet_v2.py

```diff
@@ -115,7 +117,7 @@ def resnet_v2(inputs,
               global_pool=True,
               output_stride=None,
               include_root_block=True,
-              spatial_squeeze=True,
+              spatial_squeeze=False,
```

Comment:
Yes, tested train_image_classifier.py with Resnet 50. Interestingly enough, the Resnet unit tests were failing before I applied this PR.